wiki/BBCode/en.md

```diff
@@ -1,414 +1,300 @@
+
+Within the osu! forums, code blocks are most often used to post source code for a [storyboard](/wiki/Storyboards), or in tutorials that require showing the syntax for tags, commands, or source code.
 
 ### Centre
 
-Centre align a paragraph; this is typically used for titles or headings.
+```
+[centre]text[/centre]
+```
 
-- Dedicated button: N/A
-- Notes:
-  - The tags **must** be spelt as `centre` not `center` (note the last two letters).
-- Syntax:
-  ```
-  [centre]
-  ...
-  [/centre]
-  ```
+The `[centre]` tag is used to align text to the centre of a post. This is most often used for stylistic effect in titles, headers, or poems. If placed inside of or surrounding a `[quote]` tag, the text inside the quote block will be centred, but the stylistic lines and such will not.
 
 ### URL
 
-Adds a link.
+```
+[url=LINK]text[/url]
+```
+
+The `[url]` tag is used to turn regular text into clickable hyperlinks.
 
-You do not need to use this tag if you do not want to use link text. The osu!forums will automatically link pasted URLs inside the post.
+*Note: Using this tag is not necessary should one wish not to use custom hyperlink text, as the forum editor parses proper URLs as links automatically.*
 
-- Dedicated button: ![URL button](img/url.png)
-- Notes:
-  - The `LINK` argument should not use quotation marks.
-- Syntax:
-  - Where `LINK` is the url.
-  - Where `TEXT` is the link text.
-  ```
-  [url=LINK]TEXT[/url]
-  ```
+To create hyperlinks with the `[url]` tag, users must specify two arguments: the linked text to be displayed, and the specific URL of the website to navigate to. The former must be specified between the open and closed tags, and the latter must be specified as the `LINK` argument, without quotation marks (`"`). If no text is specified, the text will default to the name of the URL.
+
+Toolbar button: ![URL button](img/url.png "URL")
 
 ### Profile
 
-Links a user profile using the user's name.
+```
+[profile=userid]username[/profile]
+```
 
-Usage of this button is **heavily discouraged**! The problem with using this tag is that user names can be changed once after obtaining an [osu!supporter tag](/wiki/osu!supporter). And once they do, the link will fail.
+The `[profile]` tag is used to link to a user's osu! profile page by using their username or user ID. User IDs are the string of numbers that directly follow the `/users/` in the URL of an osu! profile page.
 
-It is recommended to use [URL](#url) instead (using the user's id number).
+*Note: It is important to mention that the resulting outcome of this tag is functionally identical to that of using the [`[URL]`](#url) tag to link to the URL of an osu! profile page.*
```

Comment:
This is not true, using the profile tag shows a user card when hovering over the link. Links do not.